doc/nrf/gs_programming.rst

```diff
@@ -212,7 +212,11 @@ After completing the :ref:`manual <build_environment_cli>` or :ref:`automatic <g
 
       .. note::
 
-	     To build from a directory other than the application directory, run the west build command with an additional parameter *directory_name*,  specifying the application directory.
+         To build from a directory other than the application directory, run the west build command with an additional parameter *directory_name*,  specifying the application directory.
+
+         To get extra device tree overlays with new features available for a board version, run the west build command with an additional parameter *@board_revision*.
```

Comment:
What is the general guideline for multiple topics in a note? The first paragraph about directory names is not related to the board revision, so as a reader I'd prefer to have a clear distinction of the two different topics. I wouldn't read the other paragraphs if the first one was not what I was looking for. Would it be possible to make them two different notes?